File: tests/csharp/tests-csharp-xunit-allure3-playwright/Tests/Api/SeedDataApiTests.cs
using Allure.Net.Commons;
using Allure.Net.Commons.Attributes;
using Api;
using RestSharp;
using Tests;

namespace Tests.Api;

[AllureEpic("Deploy readiness")]
[AllureFeature("Seed data")]
[AllureSeverity(SeverityLevel.critical)]
[AllureSuite("Seed data on deployed stand")]
public sealed class SeedDataApiTests : ApiTestBase
{
    [Trait("TestCategory", "api")]
    [Trait("TestCategory", "smoke")]
    [Fact(DisplayName = "Flyway seed items Alpha, Beta, Gamma are present in PostgreSQL")]
    public void SeededItemsAreReadyAfterDeploy()
    {
        var response = RestSharpHttp.Request(Method.Get, "/api/items");
        Assert.True(Equals(200, response.Status), response.Body);
        Assert.Equal("postgresql", response.Text("source"));
        var names = response.ItemNames();
        Assert.True(
            names.Contains("Alpha") && names.Contains("Beta") && names.Contains("Gamma"),
            string.Join(", ", names));
    }
}

File: tests/java/tests-java-junit5-retrofit2/src/test/java/tests/api/SeedDataApiTests.java
package tests.api;

import annotations.Layer;
import api.ApiTestBase;
import api.Calls;
import api.model.Item;
import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import io.qameta.allure.Severity;
import io.qameta.allure.SeverityLevel;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

import java.util.stream.Collectors;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertTrue;

@Layer("api")
@Epic("Deploy readiness")
@Feature("Seed data")
@Severity(SeverityLevel.CRITICAL)
@DisplayName("Seed data on deployed stand")
class SeedDataApiTests extends ApiTestBase {

    @Test
    @Tag("api")
    @Tag("smoke")
    @DisplayName("Flyway seed items Alpha, Beta, Gamma are present in PostgreSQL")
    void seededItemsAreReadyAfterDeploy() {
        var response = Calls.execute(api.items());
        assertEquals(200, response.code());
        var body = response.body();
        assertNotNull(body);
        assertEquals("postgresql", body.source());
        var names = body.items().stream().map(Item::name).collect(Collectors.toSet());
        assertTrue(names.contains("Alpha"));
        assertTrue(names.contains("Beta"));
        assertTrue(names.contains("Gamma"));
    }
}
